### PR TITLE
Move WithStyleIdentifier into the correct folder

### PR DESCRIPTION
**Pull request by kerrmarin**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,6 @@ cobertura.xml
 # Python files for UITests
 *.pyc
 /.idea
+
+# The folder bundle puts its installation locally
```
Comment (deanWombourne) [inline .gitignore]: I don't understand why we need this (I don't need it on my machine)?

**Review by deanWombourne — APPROVED** [review]
ha ha ha sorry!

File: .gitignore (hunk 1 above)
Comment (kerrmarin) [reply .gitignore]: Mhm. My bad I guess. I have it set up so bundler installs all gems in `vendor` directory of your project so it's easy to package up the application for deployment.

Comment (deanWombourne) [reply .gitignore]: Ah, makes sense now :) There's probably a way in git of having device-specific git config, maybe?

**Pull request merged**